src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,4 +1,28 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/users")
 public class UserController {
+
+    @PostMapping("/createUser")
+    public GenericEntity createUser(){
+        return new GenericEntity();
+    }
+
+    @GetMapping("/accessDenied")
+    public String accessDenied(Long userId){
+        return "Access Denied.";
+    }
+
+    @GetMapping("/createKey")
```

Comment:
W wymaganiach jest informacja o odświeżaniu klucza, ale nie powinno się to robić dowolnie przez otwartą metodą controlera

Reply:
Ok, usunięte. Czy dobrze rozumiem, że takie kreowanie klucza powinno się znaleźć w ciele metody odpowiadającej za logowanie się użytkownika/pobieranie danych?